00_Base/src/config/types.ts

```diff
@@ -334,6 +341,13 @@ export const systemConfigSchema = z
           port: z.number().int().positive().optional(),
           costUpdatedInterval: z.number().int().positive().optional(),
           sendCostUpdatedOnMeterValue: z.boolean().optional(),
+          signedMeterValuesConfiguration: z
```

Comment:
1. eventually we'll move the configs into the database so we can foreign key on a specific public key config, rather than having it stored in the system config 🤞 
2. i'm not too sure myself 😳 it seems the `systemConfigInputSchema` is used to actually instantiate the config, while the `systemConfigSchema` is used just as a type 🤔

Reply:
systemConfigInputSchema has different typing around required fields because some required fields have defaults--i.e., for the input, they aren't actually required.
systemConfigSchema is the type that's actually used by the rest of the application, so all required fields are actually required in its typing.